include/dlaf/matrix/tile.h

```diff
@@ -69,17 +144,16 @@ hpx::future<Tile<T, D>> createSubTile(const hpx::shared_future<Tile<T, D>>& tile
 /// the tile of const elements ensure that the memory will not be modified.
 template <class T, Device device>
 class Tile<const T, device> {
+public:
   using TileType = Tile<T, device>;
   using ConstTileType = Tile<const T, device>;
-
-  template <class PT>
-  using promise_t = hpx::lcos::local::promise<PT>;
+  using TileDataType = internal::TileData<T, device>;
```

Comment:
shouldn't this be like this?
```suggestion
  using TileDataType = internal::TileData<const T, device>;
```

Reply:
No, as the data member of Tile and Const Tile are the same and should be modifiable.

Note that it was:
https://github.com/eth-cscs/DLA-Future/blob/master/include/dlaf/matrix/tile.h#L187